### DISCUSSION TITLE
share items with other users

### DISCUSSION THREAD

**Discussion by MrColumbo** [karakeep-app/karakeep#432] [Ideas]
Me and my wife are just doing some travel planning. Of course hoarder is nice to capture ideas and websites but how do we combine the stuff my wife captured with the stuff I captured? Is there a way to share bookmarks with other users? If not it might be a nice feature :)

**Comment by kamtschatka** [comment]
Sounds like this: https://github.com/hoarder-app/hoarder/issues/37 ?

**Reply by MrColumbo** [reply]
thanks for pointing me to #37 - somehow i did not find it
